tests/publiccloud/install_ipa.pm

```diff
@@ -65,6 +65,12 @@ sub run {
     assert_script_run("touch .config/ipa/config");
     assert_script_run("ipa list");
     assert_script_run("ipa --version");
+
+    # install azure cli
+    zypper_call('in curl');
+    assert_script_run('sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc');
+    zypper_call('addrepo --name "Azure CLI" --check https://packages.microsoft.com/yumrepos/azure-cli azure-cli');
```

Comment:
I didn't follow , before you were doing this only for azure tests and now you move it to module which got call in ec2 scenario also, right ? so do you want to add azure repo in case of ec2 test or I missing something ?

Reply:
I'm trying to prepare a system which have all tools installed. These are aws-cli, azure-cli and ipa.
I could create two + x HDD's for each single provider, but I think it doesn't make sense.

Reply:
If the plan is to create an HDD to be used as a common base of the SUT, I agree with the approach of installing all the tools in the same place. It might take longer to install, but it will done only once and reused in all the cloud providers. 
+1 to changing the name to prepare_tools before_test or whatever name that means "preparation"